### PR TITLE
pkg/cli/adm/catalog: Delegate to catalog-approvers

### PR DESCRIPTION
**Pull request by sacharya**
This command is related to building and mirroring the catalog.
Seeding the alias with ecordell for now.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -10,3 +10,5 @@ aliases:
     - smarterclayton
     - vrutkovs
     - wking
+  catalog-approvers:
+    - ecordell
```
Comment (deads2k) [inline OWNERS_ALIASES:14]: didn't even know we had this file

File: pkg/cli/admin/catalog/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+approvers:
```
Comment (deads2k) [inline pkg/cli/admin/catalog/OWNERS:4]: make him a reviewer too please.

Comment (sacharya) [reply pkg/cli/admin/catalog/OWNERS:4]: Done

File: pkg/cli/admin/catalog/OWNERS [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  - catalog-approvers
+revierers:
```
Comment (deads2k) [inline pkg/cli/admin/catalog/OWNERS:6]: typo

**Pull request merged**